demo/index.html

```diff
@@ -127,8 +136,7 @@
         getBatteryLevelButton.addEventListener('click', function() {
           batteryDevice.request().then(function(device) {
             return batteryLevel.read().then(function(value) {
-              var data = new DataView(value);
-              template.text = device.name + ' Battery Level is ' + data.getUint8(0) + '%';
+              template.text = device.name + ' Battery Level is ' + value.getUint8(0) + '%';
```

Comment:
Now that [compound bindings](https://www.polymer-project.org/1.0/docs/devguide/data-binding.html#compound-bindings) work, if you use
```
<platinum-bluetooth-device services-filter='["battery_service"]' id="battery-device" device="{{device}}">
  …
  <platinum-bluetooth-characteristic … value="{{batteryLevelData}}">
```

and assign `template.getUint8 = (value, index) => value.getUint8(0);`

you could use `<marked-element markdown="{{device.name}} Battery Level is getUint8(batteryLevelData, 0)">` to fill in the text any time the characteristic is updated.